docs/docusaurus/src/components/OverviewCard/styles.module.scss

```diff
@@ -0,0 +1,43 @@
+.overviewCard {
+  background-color: #404041;
+  color: var(--ifm-footer-link-color);
+  padding: 2.5rem 3rem 2rem;
+  position: relative;
+  width: 100%;
+  z-index: 1;
+}
+
+.overviewCard::before {
+  background: url('/img/repeating_hex_overview.png');
+  bottom: 0;
+  content: '';
+  left: 0;
+  opacity: 0.05;
+  position: absolute;
+  right: 0;
+  top: 0;
+}
+
+.overviewCard__inner_container {
+  position: relative;
+  z-index: 1;
+}
+
+.overviewCard__title {
+  max-width: 40rem;
+}
+
+.overviewCard__description {
+  font-weight: normal;
+  max-width: 40rem;
+}
+
+@media (max-width: 996px) {
+  .overviewCard {
+    padding: 3rem 2rem 2rem;
+  }
+
+  .overviewCard__title {
+    font-size: var(--h2-font-size);
```

Comment:
Do we normally applying this `--h2-font-size` variable to h1 text?

Reply:
we're not in the regular h1 tags, I guess this might have to do with the fact that the ones in particular that are inside the overview card don't have as much space as a regular h1 has in mobile, but I can ask Becca and Allen!